### ISSUE TITLE
Add option to force tiling for popup windows

### ISSUE THREAD

**Issue by KETHERCORTEX** [paperwm/PaperWM#930]
**Is your feature request related to a problem? Please describe.**
I use GIMP in multi-window mode with "canvas" window on one monitor and "tools" window on another. "Tools" window doesn't tile and I have to move it to another monitor with mouse and usually some workspace juggling. 

It's inconvenient, especially in cases when the window gets placed on monitor 1 coordinates, but the active workspace is on monitor 2, and the window ends up placed on monitor 1 **behind** that monitor's workspace. So I have to manually switch to the Tools window with alt-tab, then invoke window menu alt-space, select "Move" and drag it to monitor 1.

I'd like to describe more details about the issue, but Gnome's Looking Glass doesn't want to cooperate with me this time.

**Describe the solution you'd like**
- Add "Tile" toggle option to the window menu if the window isn't tiled by default.
- (optional) Add key combination to toggle tiling.
- (optional) Add "Untile" toggle menu option for "usual" windows that are tiled by default. It is different from "scratch". Or rather the same option as "Tile", but checked by default. It should probably also resolve the problems described in #821 to some extent.

**Describe alternatives you've considered**
I'm thinking of trying [Xpra](https://github.com/Xpra-org/xpra/), software that allows remotely display X windows on your machine. Running it inside Xwayland rootful window will make it tileable.

**Additional context**
Kids, think twice before choosing GIMP.

**Comment by Thesola10** [comment]
Oh, that would be because the tool windows are actually _popups_. PaperWM currently makes no attempt to tile popups (when it knows about them), so they aren't counted as part of the scratch layer.

As for the "tile"/"untile" action for normal windows, that would be the Scratch option. The scratch layer may bear a different name but at its core, it is unmanaged by PaperWM

**Comment by KETHERCORTEX** [comment]
> As for the "tile"/"untile" action for normal windows, that would be the Scratch option.

@Thesola10 Not necessarily. "Scratch" option places window on a layer that is present across all workspaces, but it would be useful to have windows acting like "popup" ones existing only on a single workspace.
